### PR TITLE
Support trigger by polling Artifactory

### PR DESCRIPTION
**Pull request by matankatz**
Supported since artifactory-jenkins-plugin 3.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -786,6 +786,13 @@ seed_jobs:
       pollScm: '* * * * *'
       # period trigger
       periodic: '* * * * *'
+      # Artifactory trigger
+      # To use this trigger, Artifactory should be created also by using 'Artifactory' section.
+      artifactory:
+        serverId: artName #Name of Artifactory, same as created Artifactory
+        artifactoryUrl: https://mycompany.com/artifactory #Url of Artifactory, same as created Artifactory
```
Comment (danielezer) [inline README.md]: Can't you get the url from the artifactory serverId by accessing its object? This configuration will result in specifying the artifactory url twice

Comment (matankatz) [reply README.md]: This is because the way the constructor is created in Artifactory plugin.

Comment (odavid) [reply README.md]: If you look at https://github.com/jenkinsci/artifactory-plugin/blob/master/src/main/java/org/jfrog/hudson/trigger/ArtifactoryTrigger.java#L49
You will see that it only uses the serverId (i.e. `details.getArtifactoryName()`)

Comment (matankatz) [reply README.md]: You are right, removed...

**Comment by odavid** [comment]
I am going to accept it as is, I believe we will add aliases later on, but I think it is simple like this.
Thanks again!

**Pull request merged**